lighthouse-cli/run.js

```diff
@@ -20,7 +20,7 @@ const assetSaver = require('../lighthouse-core/lib/asset-saver.js');
 
 const open = require('open');
 
-/** @typedef {import('../lighthouse-core/lib/lh-error.js')} LighthouseError */
+/** @typedef {Error & {code: string, friendlyMessage?: string}} ExitError */
```

Comment:
wasn't worth pretending this was a `LighthouseError` since either it's a regular `Error` or we construct it as just an object down on line 224 :)